configurator/rootfs/etc/configurator.conf

```diff
@@ -12,7 +12,7 @@
     "HASS_API_PASSWORD": null,
     "USERNAME": null,
     "PASSWORD": null,
-    "ALLOWED_NETWORKS": ["172.30.32.2"],
+    "ALLOWED_NETWORKS": [],
```

Comment:
This change will break Ingress support.

Could you please elaborate on the reason for this PR and the contents of this change?